admin/services/data-access/pupil-access-arrangements.data.service.js

```diff
@@ -149,19 +149,23 @@ pupilAccessArrangementsDataService.sqFindPupilsWithAccessArrangements = async (s
     }
   ]
   const sql = `
-      SELECT
-          p.urlSlug,
-          p.foreName,
-          p.middleNames,
-          p.lastName,
-          p.dateOfBirth,
-          aa.description,
-          p.checkcomplete AS hasCompletedCheck
-        FROM [mtc_admin].pupilaccessarrangements paa
-             INNER JOIN [mtc_admin].pupil p ON paa.pupil_id = p.id
-             INNER JOIN [mtc_admin].accessarrangements aa ON aa.id = paa.accessarrangements_id
-       WHERE p.school_id = @schoolId
-  `
+    SELECT * FROM (SELECT
```

Comment:
SELECT'ing from a sub-select may have a performance impact.  What does the query plan say?

Reply:
The only reason i added that was to avoid calculating the ordered by column again.  However, when running the query plan i realised i could reference the column by its alias used in the `SELECT`.

Query plan gave exact same results with and without subquery.  No doubt its working out the easiest route and optimising for us.